app/src/pages/playground/PlaygroundOutput.tsx

```diff
@@ -388,7 +536,7 @@ function PlaygroundOutputText(props: PlaygroundInstanceProps) {
     },
   });
 
-  if (instance.isRunning) {
+  if (output == null && toolCalls.length === 0) {
```

Comment:
@axiomofjoy you made this change, but it breaks streaming since the instance is still running while the chunks are coming in, it doesn't show the output till it's all done, i reverted it back but wanted to make sure the change wasn't fixing some other issue

Reply:
alternatively we can mark it as complete when the first chunk comes in but not sure if that's the right call